ods-devenv/packer/build-ods-ami-ssm.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+set -eu
+set -o pipefail
+# build-ods-ami-ssm.sh builds an ODS image using Packer over SSM.
+# It depends on a prebuilt CentOS7 AMI which has SSM installed.
```

Comment:
This line mentions that this scripts depends on a prebuild CentOS7 with SSM installed, however in the code I can't see why SSM is required. Is there a hidden dependency? If yes, can you please a line explaining this?

Reply:
It's in the packer JSON. That is configured to use SSM instead of SSH.